MANIFEST.in

```diff
@@ -1,6 +1,6 @@
 
 recursive-include Orange *.pyx *.py *.c *.cpp README* LICENSE
-recursive-include Orange/datasets *.tab *.basket *.info
+recursive-include Orange/datasets *.tab *.basket *.info *.dst
```

Comment:
I guess not, since this has not been done in the past two weeks. We can always add other formats when we introduce such datasets. We do however already have .dst file, so I am merging this PR.